src/engine/test/integration_tests/router/features/api.feature

```diff
@@ -0,0 +1,23 @@
+Feature: Router Routes API Management
+
+  @wip
+  Scenario: Add a new route to the router via API
+    Given I am authenticated with the router API "default"
+    When I send a request POST to the router to add a new route called "dummy" with the data from policy:"policy/wazuh/0" filter:"filter/allow-all/0" priority:"255"
+    Then I should receive an error response indicating that the policy already exists
+
+  Scenario: Update an existing route in the router via API
+    Given I am authenticated with the router API "default"
+    When I send a PATCH request to update the priority from route "default" to value of "250"
+    Then I should receive a success response indicating that the route was updated
+    And I should check if the new priority is 250
+
```

Comment:
It is necessary to add the test of invalid priorities, this includes invalid numbers (greater than 255) and priorities reserved for the sessions.